### DISCUSSION TITLE
想问问为啥有了极客蓝还有解密游戏

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1720] [提问] [answered]
内容都是一样的，留着有什么用吗awa

**Comment by WForst-Breeze** [comment]
个人猜测是不是为了方便进一步向下解密，可以使解密玩家回过头来分析一些线索🤔毕竟获得到极客蓝是解密游戏的第一步，这个游戏还远远没有结束

**Reply by iPodtouch6557** [reply]
这是啥意思……
~我是把那个初始信息输进搜索栏里面之后就看到隐藏主题的解锁教程了~

**Reply by WForst-Breeze** [reply, marked as answer]
现在已经允许讨论隐藏主题解锁教程了，我的意思是这个解密游戏的终点不是极客蓝，获取到极客蓝是解密游戏的第一步，还有很多谜题等待破解，解密游戏没有结束。后续的进一步解密可能需要重新分析解谜游戏前面的线索元素，所以可能就还留着你提及的解密游戏的开启通道

**Reply by iPodtouch6557** [reply]
> 现在已经允许讨论隐藏主题解锁教程了，我的意思是这个解密游戏的终点不是极客蓝，获取到极客蓝是解密游戏的第一步，还有很多谜题等待破解，解密游戏没有结束。后续的进一步解密可能需要重新分析解谜游戏前面的线索元素，所以可能就还留着你提及的解密游戏的开启通道

我就说，那个 Puzzleunlock+解锁码后四位和其他解密流程有什么关系……
就是有些未知的迷题可以再回头去解决是吧
